mod/notifications/views/default/notifications/subscriptions/record.php

```diff
@@ -25,7 +25,10 @@
 	'alt' => $entity->getDisplayName(),
 ]);
 
-$name = $entity->getDisplayName();
+$name = elgg_view('output/url', [
+	'text' => $entity->getDisplayName(),
+	'value' => $entity->getURL(),
```

Comment:
`href` not `value` don't rely on magic
Maybe also add `'is_trusted' => true`

Reply:
changed